Mathlib/GroupTheory/Submonoid/Operations.lean

```diff
@@ -1372,6 +1372,13 @@ theorem eq_bot_iff_forall : S = ⊥ ↔ ∀ x ∈ S, x = (1 : M) :=
 #align submonoid.eq_bot_iff_forall Submonoid.eq_bot_iff_forall
 #align add_submonoid.eq_bot_iff_forall AddSubmonoid.eq_bot_iff_forall
 
+@[to_additive]
+theorem eq_bot_of_subsingleton [Subsingleton S] : S = ⊥ := by
```

Comment:
Does `Subsingleton.elim _ _` not work?

Reply:
I don't understand this suggestion. Do you mean replacing the whole proof with `Subsingleton.elim _ _`? Then the answer is clearly no.

Reply:
Oh the assumption isn't `Subsingleton M`. Sorry for the noise!